docs/start.rst

```diff
@@ -84,11 +84,15 @@ Cloud Compute
 GPU Environments
 ^^^^^^^^^^^^^^^^
 
-WGPU can work in GPU cloud compute environments on Linux machines with no physical display output. By default, these environments may lack system libraries that are typically found on a standard linux desktop. On Debian & Ubuntu based systems you should be able to get everything you need by installing the following in addition to your vendor-specific (Nvidia/AMD) GPU drivers:
+WGPU can work in GPU cloud compute environments on Linux machines with no
+physical display output. By default, these environments may lack system
+libraries that are typically found on a standard linux desktop. On Debian &
+Ubuntu based systems you should be able to get everything you need by installing
+the following in addition to your vendor-specific (Nvidia/AMD) GPU drivers:
 
 .. code-block:: bash
 
-    sudo apt install xserver-xorg-core mesa-vulkan-drivers libvulkan1
+    sudo apt install -y xserver-xorg-core mesa-vulkan-drivers libvulkan1
```

Comment:
I don't think it's very security friendly to include a -y in there. People may not notice when they copy the command.